examples/example_lib_window.ks

```diff
@@ -0,0 +1,43 @@
+run lib_window_number_dialog.
+run lib_window_vessel_stats.
+run lib_window_menu.
+run lib_process.
+
+clearscreen.
+ag1 off.
```

Comment:
For debugging and demonstrating purposes, the right window can be closed by turning ag1 on (there is an info on terminal screen). To prevent itr from closing immediately, I turn ag1 off in the beginning.